tables/tests/test_enum.py

```diff
@@ -11,7 +11,7 @@
 """Test module for enumerated types under PyTables."""
 
 import operator
-import itertools
+import itertools as it
```

Comment:
also in this case I would prefer to keep "itertools"

Reply:
There are only a few lines of code, no problem.